### PR TITLE
Add DASH UK and UKRI as Funders for Fair training materials

### PR DESCRIPTION
**Pull request by khaled196**
# Thank you for contributing to the GTN! :yellow_heart:

## Please check that:

- [ ] Your pull request has a **good title**, e.g.
    - "Fix typo in ansible-galaxy tutorial"
    - "Add new transcriptomics tutorial covering a new sequencing technology"
- [ ] Please **replace this entire pull request message** with a *description of your changes*, it will help us review your pull request faster
- [ ] Check that your images are allowed to be re-hosted by the GTN

Once these are done, you're ready to go!

<details>
    <summary>Consider sustainable computing with Draft Mode :green_heart:</summary>

Is your pull request finished? 100% ready to be merged? Or do you want some time to work on it first (which we encourage! It's great to have visibility)

If not, then please consider creating this pull request as a draft.

Regular pull requests will trigger automated test runs when they are created
and every time you update them.

As a small contribution to sustainable computing, we skip the most energy intensive of these tests for pull requests marked as drafts. Once your pull request is ready to go, then you can click a button saying "Ready for Review" and the tests will be run!
</details>

### CODE REVIEW

File: CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -1885,3 +1886,22 @@ biont:
...
+
+ukri:
+    name: UKRI
+    github: false
+    joined: 2023-10
+    avatar: "https://github.com/galaxyproject/training-material/blob/main/shared/images/ukri.png?raw=true"
+    url: https://elixiruknode.org/projects/elixir-uk-dash/
+    funder: true
+    funding_statement: |
+      UK Research and Innovation (UKRI) is the national funding agency investing in science and research in the UK. Invest in research that enriches lives, drives economic growth, and creates jobs and high-quality public services.
```
Comment (hexylena) [inline CONTRIBUTORS.yaml:1898]: Just to check the relationship here, UKRI is a funding body that funds DaSH, correct? If so I think adding just the second entry would make the most sense, as we don't really annotate the funding _agencies_ themselves, but rather the specific grant that funded the work.

```suggestion
```

File: CONTRIBUTORS.yaml [hunk 2]
```diff
@@ -1885,3 +1886,22 @@ biont:
...
+
+ELIXIR-UK-DaSH:
+    name: DASH UK
+    joined: 2023-10
+    avatar: "https://github.com/galaxyproject/training-material/blob/main/shared/images/dash.png?raw=true"
+    url: https://elixiruknode.org/projects/elixir-uk-dash/
+    funder: true
```
Comment (hexylena) [inline CONTRIBUTORS.yaml:1905]:
```suggestion
    funder: true
    funding_system: ukri
    funding_id: MR/V038966/1
```

additionally is this the correct funding ID? It seems to also be used on https://training.galaxyproject.org/training-material/hall-of-fame/elixir-fair-data/ and just want to double check.

As of https://github.com/galaxyproject/training-material/pull/4408 we'll have the ability to annotate some more information about funders which could be useful, but we still need to finalise that first.

**Review by hexylena — COMMENTED** [review]
ah nice!

**Comment by kkamieniecka** [comment]
Hi @hexylena I have commit the suggestions, thanks

**Comment by hexylena** [comment]
Thanks @kkamieniecka !

File: topics/fair/tutorials/bioimage-metadata/tutorial.md [hunk 3]
```diff
@@ -34,6 +34,9 @@ contributions:
     - Laura190
     - kkamieniecka
     - poterlowicz-lab
+  funding:
+      - ELIXIR-UK-DaSH
+      - ukri
```
Comment (hexylena) [inline topics/fair/tutorials/bioimage-metadata/tutorial.md:39]:
```suggestion
```

Comment (hexylena) [reply topics/fair/tutorials/bioimage-metadata/tutorial.md:39]: could you remove it from the tutorials since we don't mention it anymore?

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Could you please remove the other `- ukri` entries now that we've removed that funder? I'd push the commit but your repo doesn't permit it

```
From f803ca4e172d84463bc24c8152c2aa5587702bdd Mon Sep 17 00:00:00 2001
From: Helena Rasche <hexylena@galaxians.org>
Date: Mon, 23 Oct 2023 16:43:07 +0200
Subject: [PATCH] fix funders

---
 topics/fair/tutorials/bioimage-REMBI/tutorial.md  | 1 -
 topics/fair/tutorials/data-management/tutorial.md | 1 -
 topics/fair/tutorials/fair-gtn/tutorial.md        | 1 -
 topics/fair/tutorials/fair-intro/tutorial.md      | 1 -
 4 files changed, 4 deletions(-)

diff --git a/topics/fair/tutorials/bioimage-REMBI/tutorial.md b/topics/fair/tutorials/bioimage-REMBI/tutorial.md
index 71f0ca339530..bea9ac5b7798 100644
--- a/topics/fair/tutorials/bioimage-REMBI/tutorial.md
+++ b/topics/fair/tutorials/bioimage-REMBI/tutorial.md
@@ -34,7 +34,6 @@ contributions:
     - poterlowicz-lab
   funding:
       - ELIXIR-UK-DaSH
-      - ukri
 
 subtopic: fair-data
 
diff --git a/topics/fair/tutorials/data-management/tutorial.md b/topics/fair/tutorials/data-management/tutorial.md
index 78c62dc6b55a..4802e8b4a114 100644
--- a/topics/fair/tutorials/data-management/tutorial.md
+++ b/topics/fair/tutorials/data-management/tutorial.md
@@ -29,7 +29,6 @@ contributions:
     - hexylena
   funding:
       - ELIXIR-UK-DaSH
-      - ukri
 subtopic: fair-data
 
 requirements:
diff --git a/topics/fair/tutorials/fair-gtn/tutorial.md b/topics/fair/tutorials/fair-gtn/tutorial.md
index 2bf44eb0e401..ca428c3d2bbe 100644
--- a/topics/fair/tutorials/fair-gtn/tutorial.md
+++ b/topics/fair/tutorials/fair-gtn/tutorial.md
@@ -28,7 +28,6 @@ contributions:
     - hexylena
   funding:
       - ELIXIR-UK-DaSH
-      - ukri
 subtopic: fair-data
 
 requirements:
diff --git a/topics/fair/tutorials/fair-intro/tutorial.md b/topics/fair/tutorials/fair-intro/tutorial.md
index 93b247cd904f..1dde75b0b4ff 100644
--- a/topics/fair/tutorials/fair-intro/tutorial.md
+++ b/topics/fair/tutorials/fair-intro/tutorial.md
@@ -28,7 +28,6 @@ contributions:
     - hexylena
   funding:
       - ELIXIR-UK-DaSH
-      - ukri
 subtopic: fair-data
 
 ---
-- 
2.40.1

```

**Review by hexylena — APPROVED** [review]

**Comment by kkamieniecka** [comment]
@hexylena, do you have any idea why the founder is not visible on GTN?

**Comment by hexylena** [comment]
we're discussing in slack but cross-posting it here, it *looks* OK to me? https://training.galaxyproject.org/training-material//topics/fair/tutorials/fair-intro/tutorial.html#funding

**Pull request merged**